### DISCUSSION TITLE
fallback repo

### DISCUSSION THREAD

**Discussion by TitaniumHocker** [artipie/artipie#1163] [Q&A] [answered]
Is there any way to specify fallback repo?
Let me explain. For example I have private repo with my private packages and I have "mirror" repo that caches packages from public repo. I want my private repo to fallback to mirror when package doesn't found.

**Comment by dgarus** [comment, marked as answer]
Hi, @TitaniumHocker!
Yes, you can create maven-proxy repo and define as the first remote a private repo and as the second a public repo.
 See [maven-proxy wiki page] (https://github.com/artipie/artipie/wiki/maven-proxy)

**Comment by dgarus** [comment]
@TitaniumHocker to get more details, you can check this [article](https://dzone.com/articles/private-remote-maven-repository-with-artipie-1)
